microshift.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+set -exuo pipefail
```

Comment:
The commit log for ` Add microshift.sh script to create VM with microshift packages ` could add a high level description of how this all works for future reference.